bindings/pyroot_experimental/pythonizations/CMakeLists.txt

```diff
@@ -92,7 +92,13 @@ foreach(val RANGE ${how_many_pythons})
 
   set(libname ROOTPythonizations${python_under_version_string})
 
-  ROOT_LINKER_LIBRARY(${libname} ${cpp_sources} DEPENDENCIES Core Tree cppyy${python_under_version_string} CMAKENOEXPORT)
+  add_library(${libname} SHARED ${cpp_sources})
+  set_target_properties(${libname} PROPERTIES  ${ROOT_LIBRARY_PROPERTIES})
```

Comment:
To be checked: why we need to set here `ROOT_LIBRARY_PROPERTIES` instead of what is done for `libcppyy` and `libcppyy_backend`, which is just `set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")`.